.gitignore

```diff
@@ -1 +1,3 @@
-/cmd/peregrine/peregrine
+cmd/peregrine/peregrine
```

Comment:
Although it doesn't matter too much in this case, it is more "correct" to have a leading slash on these entries because that will ensure that they are being matched from the root of the project.

Reply:
Ok :+1: